crates/driver/src/infra/solver/mod.rs

```diff
@@ -97,6 +99,8 @@ pub struct Config {
     pub account: ethcontract::Account,
     /// How much time to spend for each step of the solving and competition.
     pub timeouts: Timeouts,
+    //Authorization Header for requests
```

Comment:
Using `///` indicates a doc comment which will show when an IDE suggest to autocomplete the field. Also with the new system we are not limited to just supporting auth related headers.
```suggestion
    /// HTTP headers that should be added to every request.
```